### PR TITLE
Introduce K8s yaml spec for event migration tool (1091-1149).

### PR DESCRIPTION
**Pull request by aponjavic**

### CODE REVIEW

File: scripts/events/migration_1091-1149/event-migration-tool_1091-1149.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            configMapKeyRef:
+              name: sysdigcloud-config
+              key: mysql.password
+        - name: ES_ENDPOINT
+          valueFrom:
+            configMapKeyRef:
+              name: sysdigcloud-config
+              key: elasticsearch.url
+        - name: ES_PORT
+          value: "9200" # Change thi param if different
```
Comment (papajulio) [inline scripts/events/migration_1091-1149/event-migration-tool_1091-1149.yaml]: Change 'the' param if different

File: scripts/events/migration_1091-1149/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Combined Event Migration (1091 - 1149)
+
+Migration that combines events from MySQL and old ES sources into one common ES index.
```
Comment (markyjackson-taulia) [inline scripts/events/migration_1091-1149/README.md]: Please change ES to actually say elasticsearch

File: scripts/events/migration_1091-1149/event-migration-tool_1091-1149.yaml (hunk 1 above)
Comment (aponjavic) [reply scripts/events/migration_1091-1149/event-migration-tool_1091-1149.yaml]: Done

**Comment by aponjavic** [comment]
This PR should be merged ASAP (before new onprem release is officially published), since it contains necessary K8s spes for event migration tool.

cc: @mbreitung @markyjackson-taulia

**Comment by mbreitung** [comment]
these commits have been cherry picked to the release branch. See #68

**Pull request closed without merging**